### ISSUE TITLE
Flash recording not working

### ISSUE THREAD

**Issue by skrichten** [sathyapulse/angular-recorder#5]
I get this error message...

angular.js:13550 TypeError: Cannot read property 'isMicrophoneAccessible' of null
    at Object.Recorder.isMicrophoneAccessible (angular-audio-recorder.js:1154)
    at Object.swfHandlerConfig.getPermission (angular-audio-recorder.js:766)
    at Object.service.showPermission (angular-audio-recorder.js:904)
    at control.startRecord (angular-audio-recorder.js:268)
    at RecordModalController.onRecordClick (record-modal.js:33)
    at fn (eval at <anonymous> (angular.js:14432), <anonymous>:4:263)
    at expensiveCheckFn (angular.js:15485)
    at callback (angular.js:25018)
    at Scope.$eval (angular.js:17229)
    at Scope.$apply (angular.js:17329)

Also things are confusing with @logbon72 fork ....
If I run `bower install angularAudioRecorder` which one am I getting?

**Comment by sathyapulse** [comment]
@skrichten Wait for the recorder to load in the page, so that the functions can be accessible

I need to update the read me file with the logbon72 merge.

**Comment by skrichten** [comment]
What in your API will indicate when this happens?

**Comment by sathyapulse** [comment]
@skrichten FWRecorder.isReady will be true when it is ready, you can refer the index.html in the demo directory

**Comment by skrichten** [comment]
That's not working, FWRecorder.isReady is always false.  Do you have an example that uses the flash based recording?

**Comment by skrichten** [comment]
Your demo does not reference FWRecorder.isReady at all....

**Comment by skrichten** [comment]
If I set forceSwf(true) in your demo, I get the same error.

**Comment by sathyapulse** [comment]
I will check it and let you know about it.

**Comment by Luddinus** [comment]
same issue here (using safari)

**Comment by kaushik-evani** [comment]
I'm having the same issue as well and recording audio on safari is important to me. But a brilliant module though.
Any updates on this issue? Would really appreciate if you could point me towards making flash recording work on safari. Thanks. Cheers.

**Comment by kaushik-evani** [comment]
@sathyapulse @skrichten I totally agree. Its very confusing with @logbon72 repo. But I downloaded both the repos and did a quick diff. Both are the same. There is absolutely no difference at all. So I guess for now we can continue using `bower install angularAudioRecorder` untill @sathyapulse is updated.

**Comment by gabguy** [comment]
+1
